src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Extension/Roslyn.SyntaxVisualizer.Extension.csproj

```diff
@@ -15,6 +15,7 @@
     <ExtensionInstallationFolder>Microsoft\SyntaxVisualizer</ExtensionInstallationFolder>
     <VisualStudioInsertionComponent>Roslyn.SyntaxVisualizer.Extension</VisualStudioInsertionComponent>
     <Ngen>false</Ngen>
+    <UseCodeBase>true</UseCodeBase>
```

Comment:
Unrelated to this change but ` <IsProductComponent>false</IsProductComponent>` seems wrong since the project is also setting `VisualStudioInsertionComponent`. Arcade will override `IsProductComponent` if `VisualStudioInsertionComponent` is set.

Reply:
Another unrelated thing:

```
<!-- workaround for https://github.com/NuGet/Home/issues/5894 -->
  <Import Project="$(ArtifactsObjDir)$(AssemblyName)\*.props" Condition="'$(MSBuildProjectExtension)' == '.tmp_proj'" />	  <Import Project="$(ArtifactsObjDir)$(AssemblyName)\*.props" Condition="'$(MSBuildProjectExtension)' == '.tmp_proj'" />
```

```
 <!-- workaround for https://github.com/NuGet/Home/issues/5894 -->
  <Import Project="$(ArtifactsObjDir)$(AssemblyName)\*.targets" Condition="'$(MSBuildProjectExtension)' == '.tmp_proj'" />	  <Import Project="$(ArtifactsObjDir)$(AssemblyName)\*.targets" Condition="'$(MSBuildProjectExtension)' == '.tmp_proj'" />
```

shouldn't be needed and also has no effect on .NET Framework 4.7.2, since the WPF temp proj extension changed there. Arcade SDK implements a workaround for the WPF temp projects, so you should be able to just delete these.